robbot.py

```diff
@@ -26,6 +16,33 @@
 
 bot = commands.Bot(command_prefix='.')
 
+'''
+    Cog Functions & Commands
+'''
+def load_cogs():
+    for files in os.listdir('./cogs'):
+        if files.endswith('.py'):              
+            bot.load_extension(f'cogs.{files[:-3]}')
+            print(f'{files[:-3]} extension loaded')
+
+def unload_cogs():
+    for files in os.listdir('./cogs'):
+        if files.endswith('.py'):              
+            bot.unload_extension(f'cogs.{files[:-3]}')
+            print(f'{files[:-3]} extension unloaded')
```

Comment:
See comments on `load_cogs()`.

```suggestion
    for file in os.listdir('./cogs'):
        if file.endswith('.py'):              
            cog_name, file_ext = os.path.splitext(file)
            bot.unload_extension(f'cogs.{cog_name}')
            print(f'{cog_name} extension unloaded')
```